core/src/hlist.rs

```diff
@@ -324,6 +324,25 @@ macro_rules! gen_inherent_methods {
                 ToRef::to_ref(self)
             }
 
+            /// Return an HList where the contents are mutable references
+            /// to the original HList on which this method was called.
+            ///
+            /// # Examples
+            ///
+            /// ```
+            /// # #[macro_use] extern crate frunk; fn main() {
+            /// assert_eq!(hlist![].to_mut(), hlist![]);
+            ///
+            /// assert_eq!(hlist![1, true].to_mut(), hlist![&mut 1, &mut true]);
+            /// # }
+            /// ```
+            #[inline(always)]
+            pub fn to_mut<'a>(&'a mut self) -> <Self as ToMut<'a>>::Output
+                where Self: ToMut<'a>,
```

Comment:
Inconsistent style as compared to other places in this PR; use the style you've used before.